class/defaults.yml

```diff
@@ -100,10 +100,10 @@ parameters:
                 resources:
                   requests:
                     cpu: "1"
-                    memory: 8Gi
+                    memory: 4Gi
                   limits:
                     cpu: "1"
-                    memory: 8Gi
+                    memory: 4Gi
```

Comment:
This seems like a reasonable default, but at least for LPG 2 this would be too strict right? We need to remember to update the limits there and potentially on other bigger clusters

Reply:
Yes it must be pinned on LPG2 before installing the new defaults.

Reply:
Pinned